__tests__/app.test.js

```diff
@@ -247,36 +247,38 @@ describe("GET/api/articles/:article_id", () => {
         expect(body.article[0].article_id).toBe(articleId);
       });
   });
+});
+test("response with the status of 400 with apropriate response if update object is empty ", () => {
+  const update = {};
 
-  });
-  test("response with the status of 400 with apropriate response if update object is empty ", () => {
-    const update = {};
+  return request(app)
+    .patch("/api/articles/4")
+    .send(update)
+    .expect(400)
+    .then((res) => {
+      expect(res.body.message).toBe("Bad Request");
+    });
+});
+test("response with the status of 400 with apropriate response if update object is invalid type of information ", () => {
+  const update = { inc_votes: "words" };
 
-    return request(app)
-      .patch("/api/articles/4")
-      .send(update)
-      .expect(400)
-      .then((res) => {
-        expect(res.body.message).toBe("Bad Request");
-      });
-  });
-  test("response with the status of 400 with apropriate response if update object is invalid type of information ", () => {
-    const update = {inc_votes: "words"};
+  return request(app)
+    .patch("/api/articles/4")
+    .send(update)
+    .expect(400)
+    .then((res) => {
+      expect(res.body.message).toBe("Bad Request");
+    });
+});
+test(" response with 204 and deletes the comment", () => {
+  const commentIdToDelete = 1;
 
-    return request(app)
-      .patch("/api/articles/4")
-      .send(update)
-      .expect(400)
-      .then((res) => {
-        expect(res.body.message).toBe("Bad Request");
+  return request(app)
+    .delete(`/api/comments/${commentIdToDelete}`)
+    .expect(204)
+    .then(() => {
+      return removeComments(commentIdToDelete).then((result) => {
+        expect(result.rowCount).toBe(0);
       });
-  });
-
-
-
-/*
-have information to update my code
-access main database 
-update information using an update set query 
-returning a particular result 
-*/
+    });
```

Comment:
Dont forget to include error handling for this endpoint. 

What should happen when given an invalid comment_id? I.e. /api/comments/banana
What should happen when given a valid but none existent comment_id? I.e. /api/comments/9999